[Question] Is .flake8 line length used?

The black configuration makes it adjust to a line legth of 88, but looking at the [configyuration of flake8](https://github.com/OCA/oca-addons-repo-template/blob/master/.flake8#L2) I see an 80.

This is at least misleading if it is not really used. Can this be removed or clarified?

This is related to the pre-commit error seen in this manufacture pull request:

https://github.com/OCA/manufacture/pull/613/checks?check_run_id=2129342164#step:4:82

```
flake8 except __init__.py................................................Failed
- hook id: flake8
- exit code: 1

repair_stock_move/tests/test_repair_stock_move.py:73:92: B950 line too long (91 > 80 characters)
repair_stock_move/tests/test_repair_stock_move.py:75:92: B950 line too long (91 > 80 characters)
repair_stock_move/tests/test_repair_stock_move.py:91:90: B950 line too long (89 > 80 characters)
```

--- a/.flake8
+++ b/.flake8
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 80
+max-line-length = 88
 max-complexity = 16
 # B = bugbear
 # B9 = bugbear opinionated (incl line length)

